sql_joins_demystified/sql_joins_demystified.md

```diff
@@ -0,0 +1,589 @@
+<!--
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+module_template_version: 2.0.0
+language: en
+narrator: US English Female
+title: SQL Joins Demystified
+comment: Learn about SQL joins in a non-coding module.
+long_description: Usually, data in a SQL database is organized into multiple interrelated tables.  This means you will often have to bring data together from two or more tables into a single dataset to answer your research questions.  This is accomplished using JOIN commands.  This module teaches underlying data considerations and explains the type of JOINS, without going into detail about the SQL code itself.
+estimated_time: 1 hour
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Explain the significance of "one to many" data relationships and data normalization
+- Describe the various "shapes" of SQL JOIN: inner, left, right, and full
+- Explain how data can be linked between tables and define "primary keys" and "foreign keys"
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# SQL Joins
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites**  A moderate level of experience writing basic SQL code (SELECT, FROM, WHERE) and intermediate SQL code (CASE, LIKE, GROUP BY, WITH, etc.) on single tables is expected in this module. If you need to develop basic SQL fluency we recommend our module [SQL Basics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/sql_basics/sql_basics.md).  For more intermediate topics, we suggest our module [SQL Intermediate](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/sql_intermediate/sql_intermediate.md).
```

Comment:
Are CASE, LIKE, GROUP BY, WITH covered in SQL Basics, or in the Intermediate module?

It is a little unclear here. Maybe switching the order would make it clearer, like:

"These are all covered in SQL Intermediate. If you need to develop basic SQL fluency we recommend our module SQL Basics."

Reply:
I've changed the heck out of this